src/PerfView/EventViewer/EventWindow.xaml.cs

```diff
@@ -605,7 +561,143 @@ private void DoShowEventCounterGraph(object sender, ExecutedRoutedEventArgs e)
             string uri = "file:///" + html.Replace('\\', '/').Replace(" ", "%20");
             Process.Start(uri);
         }
-            
+
+        private const string PayloadToken = "Payload=\"{";
+        private const string NameToken = "Name:";
+        private const string DisplayNameToken = "DisplayName:";
+        private const string MeanToken = "Mean:";
+        private const string IncrementToken = "Increment:";
+        private const string IntervalToken = "IntervalSec:";
+
+        private Dictionary<string, List<Tuple<double, double>>> BuildCounters(EventSource source)
+        {
+            // look for events from "EventCounters"
+            // i.e. within Payload={...}, need to find Name, DisplayName and IntervalSec fields
+            // however, two counter types exist:
+            //  - Mean: Min, Max, Mean fields (all with the same current value with the 3.0 P7 implementation)
```

Comment:
FYI: 'all with same current value' is only true for PollingCounter. If you use EventCounter with more than one call to WriteMetric in a sampling interval you will observe Count > 1 and Min/Max could be different than Mean. The counters in System.Runtime don't wind up using EventCounter, but others could.

Reply:
Thanks a lot @noahfalk!
I think I've finally understood what you meant (thanks to @brianrob test code that is using EventCounter ;^)
I'll change the comments and also remove "Core" everywhere since it applies to both Framework and Core